src/Core/Content/Media/Event/MediaUploadedEvent.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Media\Event;
+
+use Shopware\Core\Framework\Context;
+use Shopware\Core\Framework\Event\EventData\EventDataCollection;
+use Shopware\Core\Framework\Event\EventData\ScalarValueType;
+use Shopware\Core\Framework\Event\FlowEventAware;
+use Symfony\Contracts\EventDispatcher\Event;
+
+/**
+ * @package content
+ */
+class MediaUploadedEvent extends Event implements FlowEventAware
```

Comment:
Do we really want to possibly start a flow whenever an image is uploaded, 
also we go in that direction we also need to build a `storer` for that event

Reply:
Do you really want to have a Flow or just an event?

Reply:
@keulinho Because our app wants to know if the shop has a new image uploaded to the store, I created this flow.
Good to know the store. Let me update and use it.

Reply:
@shyim @keulinho Good to know the `storer`, I don't know about it. Let me update to use the storer and remove the Flow

Reply:
I updated my PR to add the storer